src/tool-core/models/tool-core/architecture/interfaces/Message/Message.xtuml

```diff
@@ -14,12 +14,16 @@ INSERT INTO O_TFR
 	'',
 	"ba5eda7a-def5-0000-0000-000000000000",
 	1,
-	'// Generate the interface methods to create Port Messages.
-// An Interface mark may specify message parameters passed as JSON strings, rather than indexed array.
-// Parameter-handling for both mechanisms are built; one is then chosen based on that marked value.
+	'// Render the Interface file methods which create Port Messages.
```

Comment:
Wording is confusing.  Maybe it should be something like this:
Render methods which create instances of Port Messages.